src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,42 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private final int EMPTY = -1;
 
-    public int total = -1;
+    private final int MAX_INDEX = 11;
```

Comment:
This constant is related to MAX_AMOUNT_OF_ELEMENTS . Changing only one leads to an error (inconsistency).